src/compas_timber/connections/joint.py

```diff
@@ -155,3 +154,38 @@ def ends(self):
                 self._ends[str(beam.key)] = "end"
 
         return self._ends
+
+
+class JointOptions(object):
```

Comment:
I think this class should move to `workflow.py` as it is specific to the GH workflow. In code, you can just pass the kwargs directly to the joint class.